src/Uno.UI.Wasm/ts/Clipboard.ts

```diff
@@ -17,5 +35,25 @@
 
 			return "ok";
 		}
+
+		public static getText(requestId: string): void {
+			if (!Clipboard.dispatchGetContent) {
+				Clipboard.dispatchGetContent = (<any>Module).mono_bind_static_method("[Uno] Windows.ApplicationModel.DataTransfer.Clipboard:DispatchGetContent");
+			}
+			if (navigator.clipboard && navigator.clipboard.readText) {
+				navigator.clipboard.readText().then(
+					clipText => Clipboard.dispatchGetContent(requestId, clipText),
+					_ => Clipboard.dispatchGetContent(requestId, null));
+			} else {
+				Clipboard.dispatchGetContent(requestId, null);
+			}
+		}
+
+		private static onClipboardChanged() {
+			if (!Clipboard.dispatchContentChanged) {
+				Clipboard.dispatchContentChanged = (<any>Module).mono_bind_static_method("[Uno] Windows.ApplicationModel.DataTransfer.Clipboard:DispatchContentChanged");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5540286)